sfcasts/ep1/github-service-part-1.md

```diff
@@ -1,62 +1,69 @@
-health status and this is our expected, expected status and a string, which is the
-dinos name.
-
-All cool. We have our test set up. Let's go ahead and move to your terminal vendor,
-then PHP unit, and we have two errors. GitHub service test test get health report
-returns, correct health status for Dino with dataset sick Dino, uh, is given us an
-error because the class GitHub service is not found. Of course we're getting the same
-thing for our healthy Dino as well, which is to be expected. So let's go fix that
-back in our code. Let's come up here to our source directory, create a new folder and
-we'll call this service and inside create a new PHP class GI service. All right, our
-new service let's create our new public function, GI health report. And this of
-course accepts a string dinosaur name, and it's going to return a health status
-status. I'm going to keep saying status. All right. Now, instead the method we're
-going to first call GitHub's API. We're going to first call GitHubs, uh, API. Then
-we're going to filter the issues. And last we are always going to return a health
-status of healthy or sick. Now let's go back to our test. Oops, let's go back to our
-test and we need to add the use statement for our GitHub service. So the best way to
-do this is chop off the last couple of letters and there we go. GitHub service app
-service. That's the one we want. And PHP storm adds the use statement for us.
-
-We can go back to our terminal and let's run the test one more time. Make sure that
-yes, instead of two failures, we only have one. Our GitHub service test test GitHub
-test get health report returns, correct health status for Dino with data set sick
-Dino, it failed to asserting that two variables reference the same object. And it's
-of course doing that because we are only returning a healthy status for get health
-reports. Now that we have this kind of wired up, let's go ahead and implement the
-service.
+# GitHub Service - Part 1
 
+Now that we can see if a `Dinosaur` is accepting visitors on our dashboard, we
+need to keep the dashboard updated in real time by using the health status labels
+that GenLab has applied to several dino issues on GitHub. To do that we'll create 
+a service that will grab those labels from the issues using GitHub's API.
+
+## Test for our service first
+
+To test our new service, inside of `tests/Unit/` create a new `Service/` directory 
+and then a new class: `GithubServiceTest` which will extend `TestCase`. Add method
+`testGetHealthReportReturnsCorrectHealthStatusForDino` and inside, even
+though it doesn't exist yet, `$service = new GithubService()`.
+
+Our service will return a `HealthStatus` enum that is created from the health status
+label on GitHub, so we'll `assertSame()` that `$expectedStatus` is identical to 
+`$service->getHealthReport()`. Yes, we'll be using a data provider for this test...
+And because our service will return a *single* `HealthStatus` object, let's tell 
```

Comment:
I rephrased this section. It currently sounds like it's a big deal/explanation that we will return a SINGLE `HealthStatus` object... that thinking might be left over from the fact that we, internally, originally had this returning multiple. Check the rephrase for what I think explains this more clearly.